### PR TITLE
Update the example

### PR DESCRIPTION
**Pull request by shafreenAnfar**

### CODE REVIEW

File: examples/snowpeak/README.md [hunk 1]
```diff
@@ -326,17 +326,17 @@ service /snowpeak on new http:Listener(port) {
...
    resource function put reservations/[string id](@http:Payload rep:Reservation reservation)
                returns rep:ReservationUpdated|rep:ReservationConflict|rep:SnowpeakInternalError {}
 
    resource function delete reservations/[string id]() returns
                rep:ReservationCanceled|rep:SnowpeakInternalError {}
  
-   resource function put payments/[string id](@http:Payload rep:Payment payment)
-               returns rep:PaymentCreated|rep:PaymentConflict|rep:SnowpeakInternalError {}
+   resource function post payments/[string id](@http:Payload rep:Payment payment)
+               returns rep:PaymentReceipt|rep:PaymentConflict|rep:SnowpeakInternalError {}
```
Comment (chamil321) [inline examples/snowpeak/README.md:339]: Readme has more `ReservationCreated` usage. For example L384

**Review by chamil321 — APPROVED** [review]

File: examples/snowpeak/service/snowpeak.bal [hunk 2]
```diff
@@ -91,8 +91,8 @@ service /snowpeak on new http:Listener(port) {
             { name: "Payment", relation: "payment" }
         ]
     }
-    resource function put reservations/[string id](@http:Payload rep:Reservation reservation)
-                returns @http:Payload{mediaType: "application/json"} rep:ReservationUpdated|rep:ReservationConflict|rep:SnowpeakInternalError {
+    resource function post reservations/[string id](@http:Payload rep:Reservation reservation)
```
Comment (TharmiganK) [inline examples/snowpeak/service/snowpeak.bal:94]: This resource method change is incorrect, the `POST` resource should be the previous one

File: examples/snowpeak/service/snowpeak.bal [hunk 3]
```diff
@@ -66,13 +66,13 @@ service /snowpeak on new http:Listener(port) {
     @http:ResourceConfig {
         name: "Reservations",
         linkedTo: [
-            { name: "Reservation", relation: "edit", method: "put" },
+            { name: "Reservation", relation: "edit", method: "post" },
             { name: "Reservation", relation: "cancel", method: "delete" },
             { name: "Payment", relation: "payment" }
         ]
     }
     resource function put reservations(@http:Payload rep:Reservation reservation)
```
Comment (chamil321) [inline examples/snowpeak/service/snowpeak.bal:74]: Should this be `post`?

**Review by TharmiganK — APPROVED** [review]

File: examples/snowpeak/README.md (hunk 1 above)
Comment (shafreenAnfar) [reply examples/snowpeak/README.md:339]: Fixed

**Pull request merged**